docs/en/observability/uptime-tls-alert.asciidoc

```diff
@@ -40,14 +47,13 @@ image::images/tls-alert.png[Monitor status rule]
 [[action-types-certs]]
 == Action types
 
-You can extend your rules by connecting them to actions that use the following
+Extend your rules by connecting them to actions that use the following
 supported built-in integrations. Actions are {kib} services or integrations with
 third-party systems that run as background tasks on the {kib} server when rule conditions are met.
 
 You can configure action types on the <<configure-uptime-alert-connectors,Settings>> page.
 
-[role="screenshot"]
-image::images/alert-action-types.png[TLS certificate rule connectors]
```

Comment:
Can we delete this image? Or is it used elsewhere?

Reply:
I'm pretty sure I searched the obs project to see if it's used elsewhere, but I will double-check. The gist is that this image does not really add value beyond what users can see in the UI with their own eyes.